<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_start_games">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop_games">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stopp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>380</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Das Nim - Spiel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>380</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_game">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_winner">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause_game">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameConfig">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(253, 253, 253)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Anzahl der Spiele</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_games_num">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_player_select_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Spieler 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_player_select_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Spieler 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_p1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_p2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_clear">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Immer neu lernen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ki">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jeder Start ein neuer Tag</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_showBrain">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Matrix der AI anzeigen</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
